src/EnergyPlus/SolarShading.cc

```diff
@@ -10775,21 +10608,21 @@ namespace SolarShading {
         //  REAL(r64)    :: TransGl,ReflGl,AbsGl ! Glass layer solar transmittance, reflectance, absorptance
 
         Real64 ViewFactor;                  // temp var for view factor
-        Real64 ViewFactorTotal;             // debug var for view factor total
+//        Real64 ViewFactorTotal;             // debug var for view factor total
```

Comment:
I'm assuming you want to keep these lines for the next phase of work.

Reply:
I forgot to mention this in the documentation. Here the issue was I found that the commented variable and calculations are only used for debugging by the developers. They are not used anywhere. I wanted to removed them but was hesitating. Looks like they are more suitable to be moved to a unit test.